test/cram/interactive_viewer_dummy.ml

```diff
@@ -0,0 +1,56 @@
+open Cmdliner
+
+let input_events : Notty.Unescape.event list ref = ref []
+let output_image : Notty.image ref = ref Notty.I.empty
+
+let open_dummy_terminal () : unit =
+  let input_fd = Unix.openfile "/dev/null" Unix.[O_RDWR] 0o666 in
+  let output_fd = Unix.dup Unix.stdout in
```

Comment:
I think this is inverted. We don't want to see stdout so it's the one that should be redirected to `/dev/null`. We do want to pass things to stdin, so it's the one that should be connected to a [pipe](https://v2.ocaml.org/releases/5.1/api/Unix.html#VALpipe).